bin/tictactoe

```diff
@@ -1,3 +1,5 @@
 #!/usr/bin/env ruby
 
 require_relative '../config/environment'
+
+Game.new.start
```

Comment:
Thanks Randall! Great point. I'll make sure to keep this in mind, moving forward.